lib/shoryuken/environment_loader.rb

```diff
@@ -15,9 +15,9 @@ def initialize(options)
     end
 
     def load
-      initialize_logger
       load_rails if options[:rails]
       Shoryuken.options.merge!(config_file_options)
+      initialize_logger
```

Comment:
Oh, I'm sorry that' right. I will correct `initialize_logger` for logger in `load_rails`.